src/test/regress/sql/alter_role_propagation.sql

```diff
@@ -78,11 +78,66 @@ SELECT row(rolname, rolsuper, rolinherit,  rolcreaterole, rolcreatedb, rolcanlog
 SELECT run_command_on_workers($$SELECT row(rolname, rolsuper, rolinherit,  rolcreaterole, rolcreatedb, rolcanlogin, rolreplication, rolbypassrls, rolconnlimit, rolpassword, EXTRACT (year FROM rolvaliduntil)) FROM pg_authid WHERE rolname = 'alter_role_1'$$);
 
 
--- table belongs to a role
+-- give login permissions so that we can connect and check if the previous queries were propagated
+ALTER ROLE alter_role_1 LOGIN CONNECTION LIMIT 10;
 
 
--- we don't support propagation of configuration_parameters and notice the users
+-- alter configuration_parameter defaults for a user
 ALTER ROLE alter_role_1 SET enable_hashagg TO FALSE;
+ALTER ROLE alter_role_1 IN DATABASE template1 SET enable_hashjoin TO 0;
+
+-- as the defaults take effect on new connections, reconnect as the test user
+\c - alter_role_1
+SHOW enable_hashagg;
+SHOW enable_hashjoin;
+
+\c template1
```

Comment:
Since it's the same node this mostly tests PostgreSQL itself.

Reply:
I will try to remove such cases.

I meant to show that the config changes are successfully propagated, and are the same in all nodes. That's why I wanted to print out the configs in the coordinator, and then on worker nodes so that it is easier for the developer to compare them and see that they are indeed the same.

Reply:
removed all `\c` occurences in my tests